src/encoded/schemas/donor_specific_assembly.json

```diff
@@ -0,0 +1,265 @@
+        "category",
+        "ploidy",
+        "derived_from",
+        "software",
+        "files",
+        "number_of_chromosomes",
+        "number_of_contigs",
+        "number_of_scaffolds",
+        "genome_size",
+        "contig_n50",
+        "scaffold_n50",
+        "total_ungapped_length"
+    ],
+    "identifyingProperties": [
+        "accession",
+        "submitted_id",
+        "uuid"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/aliases"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/description"
+        },
+        {
+            "$ref": "mixins.json#/file_release"
+        },
+        {
+            "$ref": "mixins.json#/modified"
+        },
+        {
+            "$ref": "mixins.json#/schema_version"
+        },
+        {
+            "$ref": "mixins.json#/status"
+        },
+        {
+            "$ref": "mixins.json#/submitted"
+        },
+        {
+            "$ref": "mixins.json#/submitted_id"
+        },
+        {
+            "$ref": "mixins.json#/tags"
+        },
+        {
+            "$ref": "mixins.json#/uuid"
+        },
+        {
+            "$ref": "mixins.json#/version"
+        }
+    ],
+    "properties": {
+        "accession": {
+            "accessionType": "DA"
+        },
+        "schema_version": {
+            "default": "1"
+        },
+        "version": {
+            "pattern": "^[A-Za-z0-9_-.]{3,}$"
+        },
+        "submitted_id": {
+            "pattern": "^[A-Z0-9]{3,}_DONOR-SPECIFIC-ASSEMBLY_[A-Z0-9-_.]{4,}$"
+        },
+        "category": {
+            "title": "Category",
+            "description": "Category of the assembly",
+            "type": "string",
+            "enum": [
+                "Donor",
+                "Cell Line"
+            ]
+        },
+        "contig_l50": {
+            "title": "Contig L50",
+            "description": "L50 of contigs: the count of the smallest number of contigs whose total length makes up 50% of the assembly size",
+            "type": "integer",
+            "minimum": 0
+        },
+        "contig_n50": {
+            "title": "Contig N50",
+            "description": "N50 of contigs: the sequence length of the shortest contig at 50% of the total assembly length (bp)",
+            "type": "number",
+            "minimum": 0
+        },
+        "gaps_between_scaffolds": {
+            "title": "Gaps Between Scaffolds",
+            "description": "Number of gaps between scaffolds",
+            "type": "integer",
+            "minimum": 0
+        },
+        "gc_content": {
+            "title": "GC Content",
+            "description": "GC content of assembly (%)",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100
+        },
+        "genome_size": {
```

Comment:
There may be good reasons or specifications but I am curious as to when you decided to use the 'number' type and when you used 'integer' as some of the properties designated as number will be integers where floating points make no sense whereas for instance in several of the percent fields you have type as integer when decimal percentages seem possible?